shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -1462,10 +1596,17 @@ fn test_source_tls_and_single_tls() {
 #[test]
 #[serial]
 fn test_cassandra_redis_cache() {
+    let recorder = DebuggingRecorder::new();
+    let snapshotter = recorder.snapshotter();
+    let result = recorder.install();
```

Comment:
We should be able to just do: `recorder.install().unwrap();`
unwrap will internally use `{:?}` formatting so it should contain all the same information.